res/skins/LateNight/style.qss

```diff
@@ -1358,185 +1376,246 @@ QTableView:focus, QTreeView:focus {
-QTableView::indicator:unchecked { background: url(skin:/style/style_checkbox_unchecked.png);}
+/* selected items in Tree and Tracks table */
+QTreeView::item:selected,
 QTableView::item:selected {
-  color: #cfb32c;
+  color: #fff;
   background-color: #725309;
 }
 
-/* BPM lock icon in the library "BPM" column. */
-#LibraryBPMButton::indicator:checked {
-  image: url(:/images/library/ic_library_checked.png);
-}
-#LibraryBPMButton::indicator:unchecked {
-  image: url(:/images/library/ic_library_unchecked.png);
-}
-#LibraryBPMButton::item:selected {
+/* checkbox in library "Played" column */
+QTableView::indicator {
+  width: 12px;
+  height: 12px;
+  }
+  QTableView::indicator:checked {
+    background: url(skin:/style/library_checkbox_checked.png);
+  }
+  QTableView::indicator:unchecked {
+    background: url(skin:/style/library_checkbox_unchecked.png);
+  }
+
+/* BPM field in tracks table. */
+#LibraryBPMButton::item {
   color: #cfb32c;
-  background-color: #725309;
-}
+  background-color: transparent;
+  margin: 0px;
+  padding: 0px;
+  }
+  #LibraryBPMButton::item:focus {
+    color: #cfb32c;
+    background-color: #0f0f0f;
+    border: none;
+    margin: 0px;
+    padding: 0px;
+  }
+  #LibraryBPMButton::item:selected {
+    color: #fff;
+    background-color: #725309;
+    border: none;
+    margin: 0px;
+    padding: 0px;
+  }
+  #LibraryBPMSpinBox {
+    color: #fff;
+    selection-color: #0f0f0f;
+    background-color: #0f0f0f;
+    selection-background-color: #888;
+    /* This combination of negative top/bottom padding & margin
+      creates a 1px border around the spinbox' text area.
+      Increased left/right padding/margin removes the overlap of
+      text of activated spinbox and left-over text from regular
+      BPM value display. */
+    padding: -1px -8px -1px -4px;
+    margin: -2px 0px -2px -2px;
+    /* As soon as a border is defined, the up/down buttons
+      lose their style & icons
+    border: 0px;*/
+    }
+  /* Lock icon next to BPM value */
+  #LibraryBPMButton::indicator:checked {
+    image: url(:/images/library/ic_library_checked.png);
+  }
+  #LibraryBPMButton::indicator:unchecked {
+    image: url(:/images/library/ic_library_unchecked.png);
+  }
 
 /* Button in library "Preview" column */
 QPushButton#LibraryPreviewButton {
   width: 23px;
   height: 12px;
-  background: transparent;
+  background-color: #0f0f0f;
   border: 0px;
-}
-/*QPushButton#LibraryPreviewButton:focus {
-  background-color: #725309;
-}*/
-
-QPushButton#LibraryPreviewButton:!checked{
-  image: url(skin:/style/style_library_preview_play.png);
-}
-QPushButton#LibraryPreviewButton:checked{
-  image: url(skin:/style/style_library_preview_pause.png);
-}
+  margin: 0px;
+  padding: 0px;
+  }
+  QPushButton#LibraryPreviewButton:!checked{
+    image: url(:/images/library/ic_library_preview_play.png);
+  }
+  QPushButton#LibraryPreviewButton:checked{
+    image: url(:/images/library/ic_library_preview_pause.png);
+    background-color: #573F07;
```

Comment:
@daschuer  @ronso0 

> The OS theme shining though is not that much of an issue. We have this since long, so no need to fix it in a rush for 2.1.1

then removing line 1457 `background-color: #573F07;`
and changing line 1447 `background-color: #0f0f0f;` back to  `background: transparent;` should be the solution